content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: XXXX'
+date: 2021-04-08
+slug: kubernetes-1-21-release-announcement
+---
+
+**Authors:** [Kubernetes 1.21 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release_team.md)
+
+We’re pleased to announce the release of Kubernetes 1.21, our first release of 2021!
+This release consists of 50 enhancements: 15 enhancements have graduated to stable, 15 enhancements are moving to beta, and 19 enhancements are entering alpha.
+
+## Major Themes
+
+### CronJobs Graduate to Stable!
+CronJobs (previously ScheduledJobs) has been a beta feature since Kubernetes 1.8! With 1.21 we get to finally see this widely used API graduate to stable.
+
+CronJobs are meant for performing all time-related actions, namely backups, report generation and the like. Each of these tasks should be allowed to run repeatedly (once a day/month, etc.) or once at a given point in time.
+https://github.com/kubernetes/enhancements/issues/19
```

Comment:
Was this meant to be included here? The other enhancement blurbs don't have a link :x

Reply:
@salaxander / @rothgar : Please can you comment? 

@mrbobbytables : Would it make more sense to remove this or retain this & add others, as well?

Reply:
+1 to removing it or if the link is needed, it's also probably better to wrap it in a text instead of having the full link

Reply:
Sorry not meant to be included! @rothgar and I were using them to track the KEP info

Reply:
I've removed this link in the latest commit, thank you everyone!